src/stylesheets/components/_advantage.scss

```diff
@@ -1,3 +1,25 @@
+$title-font-size: 20px;
+
 .advantage {
-  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  &__icon {
+    width: 70px;
+  }
+
+  &__heading {
+    font-size: $title-font-size;
```

Comment:
You cannot access the scss variable like this, because if you initialized it inside of `_variables.scss`'s map, it becomes a css-variable, so, to get it you should use either `var(--title-font-size)` ,or scss functions `v(title-font-size)`, or `v-get(title-font-size)`.
Well, but if your variant works - that's strange.

Reply:
I was guided by your comments
// Use v(bg-color) to get var(--bg-color)
// Use v-get(bg-color) to get $bg-color